src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = 40 * days - 20;
```

Comment:
Replace the literals `40` and `20` with named constants to clarify their meaning and purpose, as per checklist item 2.